changelog.md

```diff
@@ -43,6 +43,8 @@ Saves from 8.x are not compatible with 9.0.0.
 * **[UI]** Fixed UI bug where altering an "ahead of package" TOT offset would change the offset back to a "behind pacakge" offset.
 * **[UI]** Fixed bug where changing TOT offsets could result in flight startup times that are in the past.
 * **[UI]** Fixed odd spacing of the finance window when there were not enough items to fill the page.
+* **[UI]** Fixed regression where waypoint altitude changes in the waypoint list screen are applied to the wrong waypoint.
```

Comment:
These bugs didn't exist in 8, right? We don't need to list bugs that haven't shipped here (and shouldn't because it confuses people).

Reply:
Looking at the changelog and commit history, it looks like the first item L46 was introduced in 8.1 and the second L47 was introduced in 8.0.

Reply:
Gotcha, thanks for confirming.